fishcore.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Added support for Visual Studio if a user wants to use that platform.

Reply:
Awesome! This can be great down the line if we get a license for Visual Studio.

Reply:
Luckily we can use Visual Studio Community edition. Has a lot of useful features even at that level.